mail/templates/receipt/body.html

```diff
@@ -0,0 +1,141 @@
+{% extends "email_base.html" %}
+{% load collections dollar_format countries mathfilters parse_date %}
+
+{% block styles %}
+  <style type="text/css">
+    table.receipt-table td {
+      padding: 10px;
+      vertical-align: top;
+    }
+    tr.row-dark {
+      background: #f5f5f5;
+    }
+    td.row-value {
+      color: #000000;
+    }
+    td.total-label {
+      font-weight: bold;
+      color: #000000;
+    }
+  </style>
+{% endblock %}
+
+{% block content %}
+<!-- 1 Column Text + Button : BEGIN -->
+  <tr>
+      <td style="background-color: #ffffff;">
+          <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+              <tr>
+                  <td class="row-text">
+                      <p>Dear {{ application.user.profile.name }},</p>
+                      <p>
+                        You have been accepted into the {{ application.bootcamp_run.title }} Bootcamp,
+                        {{ application.bootcamp_run.start_date|date:"F j, Y" }} - {{ application.bootcamp_run.end_date|date:"F j, Y" }}.
+                        You can access your payment statements here on your <a href="{% url "applications" %}">{{ site_name }} Dashboard</a>.
+                      </p>
+                      <p>Below you will find a copy of your receipt:</p>
+                  </td>
+              </tr>
+              <tr>
+                  <td class="row-text">
+                    <h1>Receipt</h1>
+                  </td>
+              </tr>
+              <tr>
+                <td class="row-text">
+                  <table width="100%" class="receipt-table">
+                    <tr>
+                      <td colspan="2">
+                        <h3>Order Information</h3>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Bootcamp:</td>
+                      <td class="row-value">{{ application.bootcamp_run.title }}</td>
+                    </tr>
+                    <tr>
+                      <td>Dates:</td>
+                      <td class="row-value">
+                        {{ application.bootcamp_run.start_date|parse_iso_datetime|date:"F j, Y" }} -
+                        {{ application.bootcamp_run.end_date|parse_iso_datetime|date:"F j, Y" }}</td>
+                    </tr>
+                    <tr class="row-dark">
+                      <td colspan="2">
+                        <h3>Order Information</h3>
+                      </td>
+                    </tr>
+                    {% for order in application.orders %}
+                    {% cumulative_field_total application.orders "total_price_paid" forloop.counter as running_balance %}
```

Comment:
The "Balance" is showing the cumulative amount paid at the time of each order, but I think it's supposed to show the total amount remaining that still needs to be paid.  For instance, "$3000 out of $5000" instead of "$2000".

https://projects.invisionapp.com/share/MDXEADT3GHQ#/screens/418697203_Bootcamp_Receipt_Email_Copy

Reply:
d'oh, good catch

Reply:
Balance is correct now, in design though it also includes " out of $<total_price>"

Reply:
you'll have to ignore the design a bit - it wasn't specified in the issue, but this UI should be nearly identical to the UI in `static/js/pages/PaymentHistoryPage.js`